featuregates/controller/pkg/featuregate/featuregate_controller.go

```diff
@@ -82,7 +82,7 @@ func (r *FeatureGateReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func (r *FeatureGateReconciler) toFeatureGateRequests(o client.Object) []reconcile.Request {
+func (r *FeatureGateReconciler) toFeatureGateRequests(_ client.Object) []reconcile.Request {
```

Comment:
If we're not using this parameter, it can be removed from the method signature.

Reply:
@rajathagasthya, do you think I can just delete the client.Object variable? I'm not that familiar with how these watches are registered and I think it will break:

```
// SetupWithManager sets up the controller with the Manager.
func (r *FeatureGateReconciler) SetupWithManager(mgr ctrl.Manager) error {
	return ctrl.NewControllerManagedBy(mgr).
		For(&configv1alpha1.FeatureGate{}).
		Watches(
			&source.Kind{Type: &configv1alpha1.Feature{}},
			handler.EnqueueRequestsFromMapFunc(r.toFeatureGateRequests)).
		Watches(
			&source.Kind{Type: &corev1.Namespace{}},
			handler.EnqueueRequestsFromMapFunc(r.toFeatureGateRequests)).
		Complete(r)
}
```

Reply:
I think messing with featuregate controller code is outside the scope of this PR. I only changed `o` to `_` to pass lint. To remove the parameter, I'd need to dig into the code and actually do controller tests manually. I suggest someone catch it and change it next time they work on featuregate since they'll have to do testing at that time, anyway.

Reply:
this function needs to be of type [MapFunc](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/handler/enqueue_mapped.go#L30) and it expects client.object as the parameter

Reply:
Thanks, @yharish991. I'll keep the method signature the same, then. No action.